### PR TITLE
Fix few typos

### PR DESCRIPTION
**Pull request by andriisoldatenko**
**Description of the change:**
- fix typo in CHANGELOG.md
- fix typo in doc/proposals/helm-operator.md
- fix typo in doc/proposals/ansible-operator.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -72,7 +72,7 @@
 
 ### Changed
 
-- The SDK now uses logr as the default logger to unify the logging output with the controller-runtime logs. Users can still use a logger of their own choice. See the [logging doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/logging.md) on how the SDK initializes and uses logr.
+- The SDK now uses log as the default logger to unify the logging output with the controller-runtime logs. Users can still use a logger of their own choice. See the [logging doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/logging.md) on how the SDK initializes and uses log.
```
[the remark this answers is not in the document]
Comment (andriisoldatenko) [reply CHANGELOG.md]: yes I know :) will remove it was my first thought.

[the remark this answers is not in the document]
Comment (andriisoldatenko) [reply CHANGELOG.md]: fixed.

[the remark this answers is not in the document]
Comment (estroz) [reply CHANGELOG.md]: `Copyright 2019 The Operator-SDK Authors` should exist in files that were created in 2019. If you want to check commit dates to find files created in 2019 with a 2018 license date, feel free to update those.

[the remark this answers is not in the document]
Comment (andriisoldatenko) [reply CHANGELOG.md]: good point. thanks for clarification.

**Comment by AlexNPavel** [comment]
I'm not sure. For small typos, I feel that it would be fine to update the CHANGELOG.

**Pull request merged**